### ISSUE TITLE
Skill notes

### ISSUE THREAD

**Issue by DigiBear** [zombieFox/awesomeSheet#149]
A lot of classes have abilities that have effects on skills e.g. Ranger's [Favored Enemy and Terrain], a barbarian 's [Rage] and the Slayer's [Study target]. 

Having either a section for notes underneath and/or a way to mark separate skills could greatly help in keeping track of those.

I added a photoshopped version for clarification
![skills](https://user-images.githubusercontent.com/26846158/37531476-94b93f96-293c-11e8-874b-64185afc6168.png)

of course just adding a skills note option is more than sufficient as well.

**Comment by zombieFox** [comment]
This is an interesting idea. If I understand the feature you are describing correctly this would require class specific data with a way to support the characters current "status". I've thought about building something which supports states like "blinded" or "fatigued" etc. However such a feature represents a **lot** of work. I can investigate what else this might involve. 

In the short term I've gone ahead and added a note area for the skill section.

**Comment by DigiBear** [comment]
Not necessarily. You could have the icons be capable of being applied the same way you prepare spells etc. You already have that icon system in place, all that would be needed is perhaps expansion on the total amount of icons and a way to make specific notes on the meaning of that mark. 

That last part you could probably reuse the code you made for abilities for, just let people apply an icon, a title and notescand you should have all that you need.

Adding the class specific data should be a quality of life improvement, cool but not needed.

Hope that helps!

I also found that crafts an professions only show a number behind them instead of the given name, I don't know if that was intended but it seems more convenient to me if it displays the given name. I can make a separate issue of it if you prefer.

**Comment by zombieFox** [comment]
The crafts and professions Skill Names not showing was a bug which is now fixed. Thanks for pointing that out!

I think this needs more thought on my part about how best to support conditional changes to Skill totals (and other abilities) and how to present that information on the display mode. The Pill with notes method might indeed work.

From a code perspective, the Pill module will eventually replace the Spells module once it supports the same functionality. (I've been planning to deprecate it for a while now.)

I've added a task to the Project board for now, I'm afraid this may be a long term goal for awesomeSheet.

Thanks for all the feedback and suggestions so far : )
